docker-compose.yml

```diff
@@ -44,6 +44,32 @@ services:
     depends_on:
       - mysqldb
 
+  sonarqube:
+    container_name: sonarqube
+    image: sonarqube:8-community
+    command: '-Dsonar.web.context=/sonarqube'
+    volumes:
+      - sonarqube_data:/opt/sonarqube/data
+      - sonarqube_extensions:/opt/sonarqube/extensions
+      - sonarqube_logs:/opt/sonarqube/logs
+      - sonarqube_temp:/opt/sonarqube/temp
+    ports:
+      - 9000:9000
+  sonarscanner:
```

Comment:
Can't we pass the `sonar.properties` to sonarqube container itself in volume `/opt/sonarqube/conf` rather than another container?

Reply:
Basically, `sonar.properties` most use case when we scan our code. Which handle by `sonarscanner`. So, I thought that users can change properties and make an easy scan.